package.json

```diff
@@ -37,11 +37,12 @@
     "code": "^1.3.0",
     "db-meta": "~0.4.1",
     "lab": "^5.2.1",
+    "mongodb": "^1.4.30",
     "mysql": "~2.5.4",
     "pg": "~4.2.0",
     "rimraf": "~2.2.8",
     "sqlite3": "~3.0.4",
-    "vows": "0.8.0"
+    "vows": "0.8.1"
```

Comment:
I was wondering why it was hanging... I had upped the version because of this: https://github.com/vowsjs/vows/issues/325